Makefile

```diff
@@ -43,13 +43,13 @@ ifndef GOPATH
 endif
 
 .PHONY: test
-noselinux:=$(shell mktemp /tmp/noselinux_XXXX)
-selinux:=$(shell mktemp /tmp/selinux_XXXX)
 test: check-gopath
-	grep '^func [A-Z].*' go-selinux/selinux_stub.go > ${noselinux}
-	grep '^func [A-Z].*' go-selinux/selinux_linux.go > ${selinux}
-	diff ${noselinux} ${selinux}
-	rm -f ${noselinux} ${selinux}
+	$(eval NOSELINUX := $(shell mktemp /tmp/noselinux_XXXX))
+	$(eval SELINUX := $(shell mktemp /tmp/selinux_XXXX))
+	grep '^func [A-Z].*' go-selinux/selinux_stub.go > ${NOSELINUX};
```

Comment:
It seems that `;` at EOL is redundant here (and in the lines below).

The alternative to is something like
```
diff <(grep '^func [A-Z].*' go-selinux/selinux_stub.go) <(grep ^func [A-Z].*' go-selinux/selinux_linux.go)
```
but it is bash/zsh specific

Reply:
That is a nice elegant solution.

Reply:
But getting it working in Makefile, will be difficult.

Reply:
why?
```make
checkme:
        bash -c "diff  <(grep '^func [A-Z].*' go-selinux/selinux_stub.go) <(grep '^func [A-Z].*' go-selinux/selinux_linux.go)"
```

```console
$ make  checkme
[kir@kir-rhat selinux]$ make checkme 
bash -c "diff  <(grep '^func [A-Z].*' go-selinux/selinux_stub.go) <(grep '^func [A-Z].*' go-selinux/selinux_linux.go)"
13a14
> func SetTaskLabel(label string) error {
make: *** [Makefile:35: checkme] Error 1
```

(i have deliberately removed `SetTaskLabel` for demo purposes)

Reply:
Just noticed that `.*` in grep regex is redundant

Reply:
Because I am an idiot and was trying to do this in diff command rather then launching bash.